pincer/commands/commands.py

```diff
@@ -570,7 +589,26 @@ def register_command(
 
 
 class ChatCommandHandler(metaclass=Singleton):
-    """Metaclass containing methods used to handle various commands
+    """Singleton containing methods used to handle various commands
+
+    The Register and Built_register
+    -------------------------------
+    I found the way Discord expects commands to be registered to be very different than
+    how you want to think about registering a command. ie. Discord wants nesting but we
+    don't want any nesting. Nesting makes it hard to think about commands and also will
+    increase lookup time.
+    The way this problem is avoided is by storing a version of the commands that we can
+    deal with as library developers and a version of the command that Discord thinks we
```

Comment:
Not sure if you actually meant "developers" or not :pensive: 
```suggestion
    deal with as the library develops and a version of the command that Discord thinks we
```

Reply:
This is intended. Some sacrifices had to be made to justify the monospace font.